src/map/packets_struct.hpp

```diff
@@ -5964,6 +5964,14 @@ struct PACKET_CZ_ADVENTURER_AGENCY_JOIN_RESULT {
 DEFINE_PACKET_HEADER(CZ_ADVENTURER_AGENCY_JOIN_RESULT, 0x0af8);
 #endif  // PACKETVER_MAIN_NUM >= 20191218 || PACKETVER_RE_NUM >= 20191211 || PACKETVER_ZERO_NUM >= 20191224
 
+#if PACKETVER_MAIN_NUM >= 20221005
+struct PACKET_ZC_SPECIALPOPUP {
+	int16 PacketType;
+	int32 ppId;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_SPECIALPOPUP, 0x0bbe);
+#endif
+
```

Comment:
```suggestion
```
It is already included.
This probably does not trigger a CI warning, because we do not compile any PACKETVER > 20211103 yet.